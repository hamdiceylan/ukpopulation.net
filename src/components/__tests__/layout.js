import React from "react"
import renderer from "react-test-renderer"

import Layout from "../layout"

describe("Layout", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Layout siteTitle="Uk population" />).toJSON()
    // expect(tree).toMatchSnapshot()
    expect(false).toBe(true)
  })
})
