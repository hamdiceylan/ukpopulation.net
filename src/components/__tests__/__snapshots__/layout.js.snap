// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  style={
    Object {
      "marginLeft": "auto",
      "marginRight": "auto",
      "maxWidth": "42rem",
      "padding": "2.625rem 1.3125rem",
    }
  }
>
  <header>
    <div>
      <h3
        style={
          Object {
            "fontFamily": "Montserrat, sans-serif",
            "marginTop": 0,
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "boxShadow": "none",
              "color": "inherit",
              "textDecoration": "none",
            }
          }
        />
      </h3>
      <a
        href="countries"
      >
        Countries
      </a>
      <a
        href="cities"
      >
        Cities
      </a>
      <a
        href="about"
      >
        About
      </a>
      <a
        href="contact"
      >
        Contact
      </a>
    </div>
  </header>
  <main />
  <footer>
    Â© 
    2019
     
    <a
      href="https://www.ukpopulation.net"
    >
      uk population
    </a>
  </footer>
</div>
`;
